package.json

```diff
@@ -9,7 +9,7 @@
     "dev": "npm install && webpack-dev-server --config webpack.example.config.js --hot --inline --no-info",
     "doctoc": "doctoc --maxlevel 3 README.md",
     "gh-pages": "./scripts/gh-pages.sh",
-    "lint": "eslint . --quiet --no-color",
+    "lint": "eslint . --quiet --no-color && ./node_modules/stylelint/dist/cli.js './themes/*/*.css'",
```

Comment:
No need for node_modules, everything in npm scripts will look for the binaries first in ./node_modules/.bin. SO you can just put "stylelint .."